dnd5e.css

```diff
@@ -1794,6 +1794,10 @@ h5 {
 /* ----------------------------------------- */
 /*  SRD Compendium                           */
 /* ----------------------------------------- */
+.srd-compendium.app .window-content {
+  background: url(../ui/parchment.jpg) repeat;
+  color: var(--color-text-dark-primary);
+}
```

Comment:
This needs to be changed in the LESS files (specifically `journal.less` for the SRD compendium files)